.github/qemu-docker/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM debian:bookworm
+
+WORKDIR /opt
+
+RUN apt-get -qy update && \
+	apt-get -qy \
+		-o "Dpkg::Options::=--force-confdef" \
+		-o "Dpkg::Options::=--force-confold" upgrade && \
+	apt-get -qy autoclean
+
+RUN apt-get -qy -f install \
+	bridge-utils \
+	build-essential \
+	cloud-image-utils \
+	genisoimage \
+	git \
+	htop \
+	lshw \
+	neovim \
+	openssh-server \
+	pipx \
+	procps \
+	python3-build \
+	python3-jinja2 \
+  qemu-kvm \
```

Comment:
nitty gritty, but should the indentation be the same here?

Reply:
Nice catch!